generators/app/templates/chatbot/src/bot.ts

```diff
@@ -1,16 +1,8 @@
 'use strict';
 
-// This is a temporary fix due to the way the TypeScript compiler currently functions
-// it converts single quotes to double quotes on import/require statements.
-/* tslint:disable:no-single-line-block-comment JSHint and ESLint need single line block comments */
-/* jshint quotmark:false */
```

Comment:
@beverts312  We can remove (and no longer need) this stuff anymore due to new eslint config updates